test/domain based tests/updateDexList.js

```diff
@@ -20,13 +20,23 @@ describe(scriptName, () => {
       adr.lssPauseAdmin,
       adr.lssInitialHolder,
       adr.lssBackupAdmin);
-
-    await env.lssController.connect(adr.lssAdmin).setDexList(
-      [env.lssGovernance.address, env.lssReporting.address, env.lssStaking.address], true,
-    );
   });
 
   describe('when Dex Listing an account', () => {
+    beforeEach(async () => {
+      await env.lssController.connect(adr.lssAdmin).setDexList(
+        [env.lssGovernance.address, env.lssReporting.address, env.lssStaking.address], true,
```

Comment:
But there's no test that verifies if address was added?

Reply:
![image](https://user-images.githubusercontent.com/13722088/148071744-13ff98de-f77b-40d6-9140-c9c5f52eed9a.png)